
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


  0%|                                                                                               | 0/4 [00:00<?, ?it/s]/home/users/delon/aemulusnu_massfunction/aemulusnu_massfunction/fisher_utils.py:74: RuntimeWarning: invalid value encountered in sqrt
  σlnλ = np.sqrt(σintrinsic**2 + (np.exp(lnλMean) - 1)/np.exp(2*lnλMean)) #im simplifying eq(9) of To, Krause+20, it seems like second term is small?
/home/users/delon/.local/lib/python3.9/site-packages/scipy/integrate/_quadpack_py.py:1225: IntegrationWarning: The occurrence of roundoff error is detected, which prevents 
  the requested tolerance from being achieved.  The error may be 
  underestimated.
  quad_r = quad(f, low, high, args=args, full_output=self.full_output,
  0%|                                                                                               | 0/4 [26:27<?, ?it/s]
Traceback (most recent call last):
  File "/home/users/delon/aemulusnu_massfunction/4.fisher_forecast/compute_cluster_abundance_covariance.py", line 27, in <module>
    N_fiducial = N_in_z_bins_and_richness_bins(fiducial_cosmology, richness_bin_edges, z_bin_edges)
  File "/home/users/delon/aemulusnu_massfunction/aemulusnu_massfunction/fisher_utils.py", line 139, in N_in_z_bins_and_richness_bins
    N_values[i, j] = N_in_z_and_richness_bin(lambda_min, lambda_max, z_min, z_max, cosmo=cosmo, mf = mf)
  File "/home/users/delon/aemulusnu_massfunction/aemulusnu_massfunction/fisher_utils.py", line 116, in N_in_z_and_richness_bin
    assert(error / result < .001) #.1% accurate
AssertionError
